reference/features/authentication.md

```diff
@@ -1,41 +1,45 @@
 # Authentication
 
-MeiliSearch uses a key-based authentication. There are three types of keys:
+MeiliSearch uses key-based authentication.
 
-- The **Master** key grants access to all routes.
-- The **Private** key grants access to all routes except the `/keys` routes.
+When using MeiliSearch in `production`, authentication is mandatory. You can read more about `production` mode in the [configuration reference](/reference/features/configuration).
+
+If a MeiliSearch instance does not use authentication, all routes will be publicly accessible and unprotected.
+
+## Key types
+
+MeiliSearch uses three types of keys:
+
+- The **Master** key grants access to all routes
+- The **Private** key grants access to all routes except the `/keys` routes
 - The **Public** key only grants access to the following routes:
   - `GET /indexes/:index_uid/search`
   - `POST /indexes/:index_uid/search`
   - `GET /indexes/:index_uid/documents`
   - `GET /indexes/:index_uid/documents/:doc_id`
-- Without any key, you can always access `GET /health`
-
-When a master key is provided to MeiliSearch, both the private and the public keys are automatically generated. **You cannot create any additional keys**.
 
-## Master Key
+When a master key is provided, both the private and the public keys are automatically generated. **You cannot create any additional keys**.
 
-When launching an instance, you have the option of giving a master key. By doing so, all routes will be protected and will require a key to be accessed.
+The only unprotected route available when using MeiliSearch in `production` is `GET /health`.
 
-You can specify it by passing the `MEILI_MASTER_KEY` environment variable, or using the command line argument `--master-key`.
+## Adding a key
 
-You can retrieve both the private and the public keys using the master key on the [keys route](/reference/api/keys.md).
+When launching a MeiliSearch instance, you have the option of giving it a master key. By doing so, all routes will be protected and will require a key to be accessed.
 
-#### No master key
+You can specify a key by passing the `MEILI_MASTER_KEY` environment variable, or using the command line argument `--master-key`.
 
-If no master key is provided, all routes can be accessed without requiring any key.
+After adding the master key, you can retrieve both the private and the public keys on the [keys route](/reference/api/keys.md).
 
-## API Key
+## Using an authenticated MeiliSearch instance
```

Comment:
```suggestion
## Communicate with a protected instance
```
I suggest something like this because the Meilisearch instance is not actually authenticated, it is the client that communicates with it that will authenticate itself through the header authentication key when it makes a request.

Reply:
I would say yes!